### ISSUE TITLE
Look into creating vrs digests

### ISSUE THREAD

**Issue by korikuzma** [ga4gh/vrsatile-pydantic#15]
See if it's possible to create vrs digests from pydantic models

**Comment by korikuzma** [comment]
Specifically for categorical variations. The types for _id in categorical variations are CURIEs and not GA4GH computed identifiers anymore, but would be nice to support this.

**Comment by ahwagner** [comment]
Absolutely. And they SHOULD be GA4GH computed identifiers, even if they could technically be other CURIEs too.

**Comment by korikuzma** [comment]
Moving this to [forked vrs-python repo](https://github.com/GenomicMedLab/vrs-python)
